component-overview/examples/form/InputGroup-onColoredBg.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { Datepicker } from '@sb1/ffe-datepicker-react';
+import { Dropdown } from '@sb1/ffe-dropdown-react';
+import { InputGroup, Checkbox, Input, TextArea } from '@sb1/ffe-form-react';
+import { SecondaryButton } from '@sb1/ffe-buttons-react';
+import { blueRoyal } from '@sb1/ffe-core';
```

Comment:
`blueRoyal` hører til det gamle fargeskjemaet og bør helst ikke brukes. Fargen er egentlig kun fortsatt tilgjengelig i og med at den er deprecated og ikke slettet. Det samme gjelder alle farger som _ikke_ starter på `@ffe-farge` i [colors.less](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-core/less/colors.less).

Reply:
Gikk for fargeFjell istellet